app/controllers/admin/merchant_items_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Admin::MerchantItemsController < Admin::BaseController
```

Comment:
I created a new merchant_items controller within admin. This might not be the best way to implement this behavior but it works right now.

Reply:
That's how I'd do it too.